docs/operator-docs/create-rule-in-cumulus.md

```diff
@@ -0,0 +1,37 @@
+---
+id: create-rule-in-cumulus
+title: Create Rule In Cumulus
+hide_title: true
+---
+
+# Create Rule In Cumulus
+
+Once the above files are in place and the entries created in CMR and Cumulus, we are ready to begin ingesting data. Depending on the type of ingestion (FTP/Kinesis, etc) the values below will change, but for the most part they are all similar. Rules tell Cumulus how to associate providers and collections, and when/how to start processing a workflow.
+
+1. Go To Rules Page
+
+* Go to the Cumulus dashboard, click on `Rules` in the navigation
+* Click `Add Rule`
+
+2. Complete Form
+
+* Fill out the template form. The following fields are
```

Comment:
The values are also described here: http://localhost:3000/cumulus/docs/next/data-cookbooks/setup#rules so we should reconcile that somehow so we don't have it in 2 places.

Reply:
This is version is more of a high level approach for quick reference and onboarding. The other resource goes into great detail. I would suggest possibly referring to cookbook with link if the individual wants more in depth information.

Reply:
@laurenfrederick What do you think about my earlier comment?

Reply:
This is fine. My concern is just the duplication of data that could get out of date.

Reply:
@laurenfrederick Understood. Updated content to redirect to the cookbook link and removed the quick reference list.